test/parallel/test-tls-client-getephemeralkeyinfo.js

```diff
@@ -15,9 +14,7 @@ let ntests = 0;
 let nsuccess = 0;
 
 function loadDHParam(n) {
-  let path = fixtures.fixturesDir;
-  if (n !== 'error') path += '/keys';
-  return fs.readFileSync(`${path}/dh${n}.pem`);
+  return fixtures.readKey(`dh${n}.pem`);
```

Comment:
Nit: it can also be inlined at this point. Ignore this comment if it is more readable in this way.

Reply:
I considered it but then decided to keep it this way as [test-tls-client-getephemeralkeyinfo.js](https://github.com/nodejs/node/blob/master/test/parallel/test-tls-dhe.js#L46) declares the same function with additional logic to handle the `error` case. I can still remove it if you would like.

Reply:
test-tls-dhe.js, ok it makes sense, it's really a nuance.